lib/fuzzer/InputPoolPosix.zig

```diff
@@ -0,0 +1,168 @@
+// The inputs are densly packed inside a file that all fuzzing processes have
+// mapped into memory. To know where inputs start and end, there is a second
+// file mapped by all fuzzing processes that stores the end index of every
+// stored string and some metadata (see the meta field).
+//
+// When a process finds a new good input, it (under a lock) appends to both of
+// these files. All other processes can immediately start working on this input
+//
+// We never shrink the corpus when fuzzing. One has to stop fuzzing and run a
+// separate program. That is because erasing strings would require complicated
+// synchronization between fuzzing processes. Finding good inputs seems to be
+// rare enough for the input pool to never get too large.
+
+const std = @import("std");
+const assert = std.debug.assert;
+const util = @import("util.zig");
+const check = util.check;
+const MemoryMappedList = @import("memory_mapped_list.zig").MemoryMappedList;
+
+/// maximum 2GiB of input data should be enough. 32th bit is delete flag
+pub const Index = u31;
+
+const InputPoolPosix = @This();
+
+const LatestFormatVersion: u8 = 0;
+const SignatureVersion: u32 = @bitCast([4]u8{ 'V', 'N', 'S', LatestFormatVersion });
```

Comment:
what's "VNS"? How about random bytes instead for a truly unique magic format header? or does this format match some existing format?

Reply:
replaced with random bytes